From 686aedfeba8decf53b126679266fd5849444ec87 Mon Sep 17 00:00:00 2001
From: Sergey Larin <cerg2010cerg2010@mail.ru>
Date: Sat, 9 Feb 2019 20:13:26 +0300
Subject: [PATCH] drm/panel: s6e63m0: Increase back porch

Original timings cause issues on Tegra hardware - image is
shifted and colors are wrong.

Signed-off-by: Sergey Larin <cerg2010cerg2010@mail.ru>
---
 drivers/gpu/drm/panel/panel-samsung-s6e63m0.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/panel/panel-samsung-s6e63m0.c b/drivers/gpu/drm/panel/panel-samsung-s6e63m0.c
index 3c84f3c073ae..db2c34adf26d 100644
--- a/drivers/gpu/drm/panel/panel-samsung-s6e63m0.c
+++ b/drivers/gpu/drm/panel/panel-samsung-s6e63m0.c
@@ -114,9 +114,9 @@ static const struct drm_display_mode default_mode = {
 	.hsync_end	= 480 + 16 + 2,
 	.htotal		= 480 + 16 + 2 + 16,
 	.vdisplay	= 800,
-	.vsync_start	= 800 + 28,
-	.vsync_end	= 800 + 28 + 2,
-	.vtotal		= 800 + 28 + 2 + 1,
+	.vsync_start	= 800 + 27,
+	.vsync_end	= 800 + 27 + 2,
+	.vtotal		= 800 + 27 + 2 + 2,
 	.vrefresh	= 60,
 	.width_mm	= 53,
 	.height_mm	= 89,
-- 
2.22.0

